hal/armv7m/imxrt/117x/peripherals.h

```diff
@@ -66,15 +66,20 @@
 #endif
 
 #ifndef UART11
-#define UART11 1
+#define UART11 0
 #endif
 
 #ifndef UART12
-#define UART12 1
+#define UART12 0
+#endif
+
+#ifdef UART_CONSOLE_PLO
+#undef UART_CONSOLE
+#define UART_CONSOLE UART_CONSOLE_PLO
```

Comment:
Perhaps plo should use `UART_CONSOLE_PLO` instead and here do
```
#ifndef UART_CONSOLE_PLO
#define UART_CONSOLE_PLO UART_CONSOLE
#endif
```

Reply:
I see `undef` is no-go here, but it's the same as in imxrt1064:
https://github.com/phoenix-rtos/plo/blob/dc68030b9ee8cc21eab861b6b274368fa31dddf1/hal/armv7m/imxrt/10xx/peripherals.h#L91-L100

I agree with your suggestion. I can change this for imxrt 1064 as well but it requires replacing all occurrences of `UART_CONSOLE` with `UART_CONSOLE_PLO` in [`console.c`](https://github.com/phoenix-rtos/plo/blob/master/hal/armv7m/imxrt/10xx/console.c) implementation - for me it is ok - I may do it if you agree ?

Reply:
I don't mean it as no-go, I just think different approach seems better.

This approach with `UART_CONSOLE_PLO` is used on zynq target already, so I see nothing of controversy here, if you're willing to make the changes I'd be grateful ;)